tests/Doctrine/Tests/DBAL/Functional/LastInsertIdTest.php

```diff
@@ -29,7 +29,7 @@ protected function tearDown()
             $this->_conn->getSchemaManager()->dropTable('last_insert_id_table');
         }
 
-        return parent::tearDown();
+        parent::tearDown();
```

Comment:
`setUp` and `tearDown` should be marked as `void` in here, and we should add type hints too (doing a sweep for low hanging issues with the patch)